cordial_dialogue/scripts/dialogue.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+from sys import byteorder
+from array import array
+from struct import pack
+import sys
+import pyaudio
+import wave
+import struct
+import rospy
+import actionlib
+from lex_common_msgs.srv import *
+from std_msgs.msg import String
+from audio_common_msgs.msg import AudioData
+from cordial_manager.msg import *
+
+PROMPT_MESSAGE = ''
```

Comment:
maybe not necessarily for this one, but we might want globals on rosparam server so other scripts can see them, or have them as part of the dialogue server